packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -4603,56 +4619,69 @@ export function attach(
         const prevContentFiber = prevFiber.child;
         const nextContentFiber = nextFiber.child;
         if (nextContentFiber === null || prevContentFiber === null) {
-          throw new Error(
-            'There should always be an Offscreen Fiber child in a Suspense boundary.',
+          const previousSuspenseState = prevFiber.memoizedState;
+          const nextSuspenseState = nextFiber.memoizedState;
+          if (
+            previousSuspenseState === null ||
+            previousSuspenseState.dehydrated === null ||
```

Comment:
If the previous one is dehydrated but the next one is not, then it's not dehydrated anymore. Seems like it shouldn't go down this path. It needs to add the hydrated children.